### PR TITLE
Add herbs2rest

### PR DESCRIPTION
**Pull request by gabrielmellooliveira**

### CODE REVIEW

File: docs/glues/herbs2rest.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+const routes = new express.Router()
+
+generateRoutes(controllerList, routes)
+
+app.use(routes)
+```
+
+#### Authorization
+
+All usecases must implement the authorize method and receive a user to authenticate.
```
Comment (dalssoft) [inline docs/glues/herbs2rest.md:64]: > "All usecases must implement the authorize method and receive a user to authenticate "

`must`?

File: docs/glues/herbs2rest.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    getAll: require('../usecases/getLists'),
+    getById: require('../usecases/getLists'),
+    post: require('../usecases/createList'),
+    put: require('../usecases/updateList'),
+    delete: require('../usecases/deleteList')
+  }
+]
+```
+
+The name field is the name of the route.
```
Comment (dalssoft) [inline docs/glues/herbs2rest.md:40]: highlight variable names in the text:
The `name` field is the name of the route.

Also, give examples: 
The `name` field is the name of the route. Ex: `https://example.com/lists/...`

**Review by dalssoft — CHANGES_REQUESTED** [review]

File: docs/glues/herbs2rest.md (hunk 1 above)
Comment (gabrielmellooliveira) [reply docs/glues/herbs2rest.md]: Revised

File: docs/glues/herbs2rest.md (hunk 2 above)
Comment (gabrielmellooliveira) [reply docs/glues/herbs2rest.md]: Revised

**Comment by jhomarolo** [comment]
Hi @gabrielmellooliveira your PR is breaking because we've changed the docusaurus version.

**Comment by gabrielmellooliveira** [comment]
@jhomarolo some links were broken, I fixed

**Pull request merged**